src/dos/drive_local.cpp

```diff
@@ -553,6 +554,9 @@ Bits localDrive::UnMount()
 	return 0;
 }
 
+void diskio_delay(Bits value /*bytes*/, DiskNoiseDevice* disknoise,
```

Comment:
`value` of what? The name needs to be descriptive and include unit of measurement.